package.json

```diff
@@ -1,158 +1,132 @@
-		"@carbon/icons-angular": "11.0.0",
-		"@commitlint/cli": "8.3.5",
-		"@commitlint/config-conventional": "8.3.4",
-		"@compodoc/compodoc": "1.1.7",
-		"@storybook/addon-actions": "5.3.13",
-		"@storybook/addon-knobs": "5.3.13",
-		"@storybook/addon-links": "5.3.13",
-		"@storybook/addon-notes": "5.3.13",
-		"@storybook/addon-options": "5.3.13",
-		"@storybook/addon-storysource": "5.3.13",
-		"@storybook/addons": "5.3.13",
-		"@storybook/angular": "5.3.13",
-		"@types/jasmine": "2.8.2",
-		"@types/node": "11.13.0",
-		"angular2-template-loader": "0.6.2",
-		"autoprefixer": "8.6.5",
-		"babel-loader": "8.0.5",
-		"carbon-components": "10.11.2",
-		"chai": "4.2.0",
-		"codelyzer": "5.0.0",
-		"commitizen": "4.0.3",
-		"core-js": "2.6.11",
-		"css-loader": "2.1.1",
-		"cz-conventional-changelog-components": "1.0.0",
-		"element-closest-polyfill": "1.0.0",
-		"exported-tests": "1.0.1",
-		"extract-text-webpack-plugin": "3.0.2",
-		"file-loader": "3.0.1",
-		"file-type": "12.4.0",
-		"gulp": "4.0.0",
-		"gulp-concat": "2.6.1",
-		"gulp-tap": "1.0.1",
-		"html-loader": "0.5.5",
-		"html-webpack-plugin": "3.2.0",
-		"husky": "1.3.1",
-		"jasmine-core": "3.4.0",
-		"jasmine-spec-reporter": "4.2.1",
-		"karma": "3.1.1",
-		"karma-chrome-launcher": "2.2.0",
-		"karma-cli": "2.0.0",
-		"karma-coverage-istanbul-reporter": "2.0.5",
-		"karma-jasmine": "2.0.1",
-		"karma-jasmine-html-reporter": "1.4.2",
-		"lodash-es": "4.17.15",
-		"ng-packagr": "4.7.1",
-		"postcss-loader": "3.0.0",
-		"raw-loader": "1.0.0",
-		"rxjs": "6.4.0",
-		"sass": "1.22.12",
-		"sass-loader": "7.1.0",
-		"semantic-release": "17.0.4",
-		"style-loader": "0.23.1",
-		"stylelint": "8.4.0",
-		"stylelint-webpack-plugin": "0.10.5",
-		"svgxuse": "1.2.6",
-		"ts-helpers": "1.1.2",
-		"ts-loader": "5.3.3",
-		"tsickle": "0.35.0",
-		"tslib": "1.9.3",
-		"tslint": "5.15.0",
-		"tslint-loader": "3.5.4",
-		"typescript": "3.2.4",
-		"wait-on": "3.2.0",
-		"webpack": "4.29.6",
-		"webpack-cli": "3.3.0",
-		"webpack-dev-server": "3.2.1",
-		"webpack-node-externals": "1.7.2",
-		"whatwg-fetch": "3.0.0",
-		"zone.js": "0.8.26"
-	},
-	"dependencies": {
-		"@carbon/utils-position": "1.1.3",
-		"flatpickr": "4.6.1"
-	}
+  "name": "carbon-components-angular",
+  "version": "0.0.0",
+  "description": "Next generation components",
+  "main": "index.js",
+  "scripts": {
+    "build": "bash scripts/build.sh",
+    "storybook": "start-storybook -s .storybook/public -p 6006",
+    "docs:build": "compodoc -p tsconfig.json",
+    "docs:server": "compodoc -p tsconfig.json -s -w",
+    "lint": "tslint 'src/**/*.ts'",
+    "lint:fix": "tslint --fix 'src/**/*.ts'",
+    "test": "ng test --no-watch",
+    "test:watch": "ng test --watch",
+    "build-storybook": "build-storybook -c .storybook -s .storybook/public -o dist/docs/storybook",
+    "semantic-release": "semantic-release",
+    "commit": "git-cz",
+    "ng": "ng",
+    "ng:build": "node --max_old_space_size=4096 ./node_modules/@angular/cli/bin/ng build"
```

Comment:
Do you still need the `--max_old_space_size=4096` if you upgrade the version of node to 12?

Reply:
I _think_ so ... I'll double check though

Reply:
I'm going to leave it as is, just so we have some room in the future if something decides it's going to eat up a ton of memory ... it doesn't seem to hurt anything as is